aas_core_meta/v3.py

```diff
@@ -1,5 +1,11 @@
 """
-Provide the meta-model for Asset Administration Shell V3.0.
+The presented version of the Metamodel within this documentation
+is related to the work of aas-core-works, which can be found here.
+The presented content of this website is neither related to the IDTA nor
+Plattform Industrie 4.0 and does not represent an official document.
+
+WIP Content of Metamodel with Version v3.0.
+Any static content or specific content for this model version can be presented here.
 
 We had to diverge from the book in the following points.
```

Comment:
Please name the points or remove sentence.

Reply:
I'd remove the sentence. Bear in mind that it makes no sense in a context of an SDK.

Reply:
See S59c1702a0ef8809115c3eaa8823a0a1dc3435726

Reply:
Thanks, LGTM!